setup.py

```diff
@@ -85,7 +85,9 @@ def _core_version(plugin_version: str = _plugin_version()) -> str:
         f"dbt-core~={_core_version()}",
         f"dbt-postgres~={_core_version()}",
         "boto3~=1.26.26",
-        "redshift-connector~=2.0.911",
+        # dbt-core uses these packages deeply, throughout the codebase, and there have been breaking changes in past patch releases (even though these are major-version-one).
```

Comment:
> dbt-core
Technically this package is used by `dbt-redshift`, not `dbt-core`. It's semantics, but should we consider updating this here?